Document high level roles and functions of the nodes making up the cn #731 (#1692)
hyperledger-labs/splice: docs/src/overview/overview.rst

- Its native token, Canton Coin, is a utility token launched as part of the Global Synchronizer. Designed to overcome enterprise adoption barriers of other networks and be compatible with an ecosystem used by regulated financial enterprises.

If you need more details on any particular mechanism (such as fee calculations, activity record structure, or minting rounds), the following white papers provides further technical specifications: `Canton Network white paper <https://www.digitalasset.com/hubfs/Canton/Canton%20Network%20-%20White%20Paper.pdf>`_, `Canton Coin white paper <https://www.digitalasset.com/hubfs/Canton%20Network%20Files/Documents%20(whitepapers%2c%20etc...)/Canton%20Coin_%20A%20Canton-Network-native%20payment%20application.pdf>`_.

Global Synchronizer
###########################

The Global Synchronizer is a decentrally operated service, using a 2/3 majority Byzantine Fault Tolerant (:term:`BFT`) consensus protocol for message ordering and confirmation, and BFT majority voting on governance changes.

> Its infrastructure is operated by independently acting organizations that run components of the decentralized infrastructure called Super Validators, that coordinate activities via an on-chain governance application.
> Its open source code is maintained in `Splice <https://github.com/hyperledger-labs/splice>`__.

Reviewer: ```suggestion
Its infrastructure is operated by independently acting organizations, called Super Validators, that run components of the decentralized infrastructure, and coordinate activities via an on-chain governance application.
Its open source code is maintained in `Splice <https://github.com/hyperledger-labs/splice>`__.
```

> The `Global Synchronizer Foundation ("GSF") <https://sync.global>`__ has been created in partnership with the Linux Foundation to coordinate the governance of the Global Synchronizer and lead efforts in growing the Global Synchronizer ecosystem.
> The GSF provides transparency into Super Validator operations and has a node in the Global Synchronizer to take part in votes on behalf of its members.

Reviewer: ```suggestion
The `Global Synchronizer Foundation ("GSF") <https://sync.global>`__ has been created in partnership with the Linux Foundation to coordinate the governance of the Global Synchronizer and lead efforts to grow the Global Synchronizer ecosystem.
The GSF provides transparency into Super Validator governance and operations. The GSF also operates a Super Validator node, and takes part in governance votes on behalf of its members.
```

Super Validators
###########################

Super Validators form the backbone of the Global Synchronizer's decentralized interoperability and synchronization infrastructure. They ensure the integrity, security, and operational reliability of the Global Synchronizer by:

- Running the core infrastructure of the Global Synchronizer
- Sequencing transactions across the network
- Validating Canton Coin transactions
- Participating in network governance and decision-making

Validators
###########################

Validators work within the broader Canton Network—a “network of networks” where each node stores only the data it needs and synchronizes with other Validators via synchronizers. Validators typically connect to one or more of these synchronizers (which might be run as centralized or decentralized services) to receive and confirm encrypted messages.

A Validator in the Global Synchronizer ecosystem primary roles are to:

* Validate transactions
* Record activity

> * Facilitate network connectivity

Reviewer: ```suggestion
* Facilitate network connectivity for users and applications
```
